sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -3170,15 +3170,28 @@ pi_result piDeviceGetInfo(pi_device Device, pi_device_info ParamName,
     // Only report device memory which zeMemAllocDevice can allocate from.
     // Currently this is only the one enumerated with ordinal 0.
```

Comment:
It looks like this behavior will be broken with these changes.
I mean that with these changes we again just sum all types of memory - global, local, grf.

Reply:
The zesDeviceEnumMemoryModules returns memories from different sub-devices, hence the sum.